src/Middleware/RoutingMiddleware.php

```diff
@@ -11,13 +13,13 @@
  */
 class RoutingMiddleware
 {
-    public function __invoke($request, $response, $next)
+    public function __invoke(ServerRequestInterface $request, ResponseInterface $response, $next)
     {
         try {
             $params = (array)$request->getAttribute('params', []);
             if (empty($params['controller'])) {
                 $path = $request->getUri()->getPath();
-                $request = $request->withAttribute('params', Router::parse($path, $request->getMethod()));
+                $request = $request->withAttribute('params', Router::parse($path));
```

Comment:
This line should not be changed. In `3.next` `Router::parse()`supports 2 arguments.